### ISSUE TITLE
Cannot load GUI

### ISSUE THREAD

**Issue by johnsills1** [oznu/homebridge-config-ui-x#179]
I have had this working before, but now I'm stuck

I recently deleted and reinstalled homebridge on docker using a Synology NAS (again) no issues

I install a simple plugin so that the library isn't empty (homebridge-marantz-volume) (no issues - it works using Homekit to control the Denon receiver volume/on & off controls)

As soon as I install the Homebridge-config-ui-x plugin, I change the config.json file to incorporate the platform entry described on the github site. "platform": "config" and so forth

After that, I try to restart the homebridge and get an error that the config entry is not registered by any plugin or whatever that error said, can't remember exactly. I just know if I add the config.json file changes i get an error.

I did not have that change on my previous setup. I could just type localhost:8581 in a web browser and the gui would come up. I've tried localhost:8080 til i'm blue in the face. Nothing works. Please help.

**Comment by IronButterfly** [comment]
The same error about unknown 'config' plugin or something like that after adding lines to config.json

**Comment by IronButterfly** [comment]
Error log:

`crond[298]: crond (busybox 1.28.4) started, log level 5                                                                                                                                                                                                                                                                        
-U /homebridge -P /homebridge/node_modules -C                                                                                                                                                                                                                                                                                  
[11/20/2018, 1:28:59 PM] Loaded config.json with 0 accessories and 2 platforms.                                                                                                                                                                                                                                                
[11/20/2018, 1:28:59 PM] ---                                                                                                                                                                                                                                                                                                   
[11/20/2018, 1:28:59 PM] Loaded plugin: homebridge-dummy                                                                                                                                                                                                                                                                       
[11/20/2018, 1:28:59 PM] Registering accessory 'homebridge-dummy.DummySwitch'                                                                                                                                                                                                                                                  
[11/20/2018, 1:28:59 PM] ---                                                                                                                                                                                                                                                                                                   
[11/20/2018, 1:29:00 PM] Loaded plugin: homebridge-mi-aqara                                                                                                                                                                                                                                                                    
[11/20/2018, 1:29:00 PM] Registering platform 'homebridge-mi-aqara.MiAqaraPlatform'                                                                                                                                                                                                                                            
[11/20/2018, 1:29:00 PM] ---                                                                                                                                                                                                                                                                                                   
[11/20/2018, 1:29:00 PM] Loading 2 platforms...                                                                                                                                                                                                                                                                                
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] Initializing MiAqaraPlatform platform...                                                                                                                                                                                                                                            
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]**************************************************************                                                                                                                                                                                                                
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]           MiAqaraPlatform v0.7.3 By YinHang                                                                                                                                                                                                                                  
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]  GitHub: https://github.com/YinHangCode/homebridge-mi-aqara                                                                                                                                                                                                                  
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]                                         QQ Group: 107927710                                                                                                                                                                                                                  
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]**************************************************************                                                                                                                                                                                                                
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]start success...                                                                                                                                                                                                                                                              
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]config gateways: sid                                                                                                                                                                                                                                                          
[11/20/2018, 1:29:00 PM] [MiAqaraPlatform] [INFO]binding to the default interface                                                                                                                                                                                                                                              
/usr/local/lib/node_modules/homebridge/lib/api.js:122                                                                                                                                                                                                                                                                          
      throw new Error("The requested platform '" + name + "' was not registered by any plugin.");                                                                                                                                                                                                                              
      ^                                                                                                                                                                                                                                                                                                                        
                                                                                                                                                                                                                                                                                                                               
Error: The requested platform 'config' was not registered by any plugin.                                                                                                                                                                                                                                                       
    at API.platform (/usr/local/lib/node_modules/homebridge/lib/api.js:122:13)                                                                                                                                                                                                                                                 
    at Server._loadPlatforms (/usr/local/lib/node_modules/homebridge/lib/server.js:316:45)                                                                                                                                                                                                                                     
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:86:36)                                                                                                                                                                                                                                                 
    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:45:10)                                                                                                                                                                                                                                                
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)                                                                                                                                                                                                                                        
    at Module._compile (internal/modules/cjs/loader.js:688:30)                                                                                                                                                                                                                                                                 
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)                                                                                                                                                                                                                                                   
    at Module.load (internal/modules/cjs/loader.js:598:32)                                                                                                                                                                                                                                                                     
    at tryModuleLoad (internal/modules/cjs/loader.js:537:12)                                                                                                                                                                                                                                                                   
    at Function.Module._load (internal/modules/cjs/loader.js:529:3)`

config.json:

`    "bridge": {
        "name": "Homebridge",
        "username": "CC:22:3D:E3:CE:30",
        "port": 51826,
        "pin": "031-45-154"
    },

    "description": "This is an example configuration file. You can use this as a template for creating your own configuration file containing devices you actually own.",

    "accessories": [
    ],


    "platforms": [{
    "platform": "MiAqaraPlatform",
    "gateways": {
    "sid": "password"
    }
    },
    {
      "platform": "config",
      "name": "Config",
      "port": 8083,
      "sudo": true
    }
    ]
    }
`

**Comment by IronButterfly** [comment]
@johnsills1 thank you! But I use Qnap NAS and Container Station.

**Comment by oznu** [comment]
Are you using the oznu/homebridge docker container?

https://github.com/oznu/homebridge-config-ui-x/wiki/Enabling-UI-with-Docker

If so all you need to do is set these env vars:

```
HOMEBRIDGE_CONFIG_UI=1
HOMEBRIDGE_CONFIG_UI_PORT=8080
```

Do not put any homebridge-config-ui-x related config in your config.json to start with. The docker container will set it up using the correct settings if these environment variables are set.

**Comment by johnsills1** [comment]
Oznu, I am getting an error about duplicate UUID. Can you tell me how to fix that situation. I am trying to install Chamberlain plugin and get that error.

**Comment by oznu** [comment]
Hi @johnsills1,

If you're still facing this issue could you post a full log output? The duplicate UUID problem is caused by a badly behaving plugin.

**Comment by johnsills1** [comment]
Well, actually, I don’t really know what I did to fix that problem, but now when I get the plugin installed, it gets stuck showing the door in the opening state in the home app. If I press the button to close the door, nothing happens to the door, it just gets stuck showing the door is closing.  At this point, I have talked to Brandon Martinez about the issue to no avail. I think it’s an API problem.
